galsim_extra/wide_scattered.py

```diff
@@ -114,6 +114,7 @@ def buildImage(self, config, base, image_num, obj_num, logger):
         #print('config = ',galsim.config.CleanConfig(config))
         for k in range(self.nobjects):
             base['obj_num'] = obj_num + k
+            obj_type, _ = galsim.config.ParseValue(base["stamp"], "obj_type", base, str)  # manually computing this ensures that we have @stamp.obj_type available for computing appropriate world_pos in the image
```

Comment:
First, I don't see why this would be required.  @stamp.obj_type will do the parsing if it's not yet been parsed.  So I don't think this should be required.

Second, this change is implicitly adding a dependency of the MixedScene stamp type on the WideScattered image type.  Maybe that's what you want, but that kind of coupling is generally not a good idea.  More specifically, it means people couldn't use WideScattered with any stamp type that didn't have an 'obj_type' parameter.

What is the @stamp.obj_type currently doing wrong without this line?  Is it crashing?  Or returning the wrong thing?

Reply:
It doesn't crash, but I'm not entirely sure what's happening with it. As seen in the images I posted above, we just lose all the galaxies for some reason. I will do some breakpoint testing later this evening

Reply:
OK, definitely try to understand what is actually going wrong currently.  I suspect this isn't the right fix to whatever the problem is though.

Reply:
Yes, definitely agree that this proposed change is more of a triage than an actual solution!

I haven't come to an understanding of why this helps, but I have found that the stamp setup doesn't happen until after this `world_pos` is computed. I don't know if this due to the order of how GalSim processes a config (image, then stamp?) or due to how these are being composed within eastlake.

Reply:
I'd make an example without Eastlake in pure galsim and debug that.